client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -189,7 +194,9 @@ const BrewRenderer = createClass({
 
 						<div className='pages' ref='pages'>
 							{/* Apply CSS from Style tab */}
-							<div style={{ display: 'none' }} dangerouslySetInnerHTML={{ __html: `<style> ${this.props.style} </style>` }} />
+							{this.state.isMounted
```

Comment:
Can we combine this check with the identical one below? Something like this? (my syntax might be slightly off)

```
{this.state.isMounted &&
  <>
    {this.renderStyle()} 
    {this.renderPages()}
  </>
}

Reply:
Checks are now combined; IMO your logic here is better than what was in the code, so I've changed it.